larq_compute_engine/core/bgemm_kernels_arm.h

```diff
@@ -49,6 +49,30 @@ struct BgemmKernel<ruy::Path::kNeonDotprod, LhsScalar, RhsScalar, DstScalar,
     TFLITE_DCHECK(false);
   }
 };
+#if RUY_PLATFORM_NEON && RUY_OPT(ASM) && RUY_PLATFORM_NEON_32
```

Comment:
```suggestion

#if RUY_PLATFORM_NEON && RUY_OPT(ASM) && RUY_PLATFORM_NEON_32
```

This suggestion just adds a blank line above.